canary/e2e/cypress/integration/common/shared.ts

```diff
@@ -0,0 +1,276 @@
+/// <reference types="@testing-library/cypress" />
```

Comment:
Can we share these files rather than duplicating them over from the e2e package? There's a lot of unused code in these files.

Reply:
I don't think that's possible. I feel a few shared command files are OK.

Reply:
**Assuming** we will separate canaries outside GH actions (or to a new repo?), I'm okay with having these separated. But definitely up to debate since it's a lot of duplicated files.

Reply:
I went ahead and deleted a bunch of the command files, and cleaned up this shared folder, so there is only a few duplicates.

Reply:
Updated here 6841e8bee15b0d93ce68fe4696422b9bd28de384